qemu/tests/cpu_flag_intel_pt.py

```diff
@@ -0,0 +1,83 @@
+import logging
+
+from avocado.utils import cpu, process
+from virttest import error_context, env_process
+from qemu.tests.x86_cpu_model import check_flags
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test cpu flag intel-pt.
+    1) Check if current flags are in the supported lists, if no, cancel test
+    2) Otherwise, set pt_model = 1 on host at first
+    3) For q35
+       3.1) boot guest with cpu model 'host' without intel-pt
+       3.2) boot guest with cpu model 'Icelake-Server' with intel-pt
```

Comment:
I guess you can use auto cpu model for this scenario,

Reply:
> I guess you can use auto cpu model for this scenario,

Is there possible that the auto model is duplicate with 'host'?

Reply:
No, the auto selected models are listed in cpu.py in vt, which doesn't include 'host'.

Reply:
Thanks, will separate one new patch for the flag `intel-pt`  since it needs more time to investigate.